plugins/workbench-resources/src/components/SpecialView.svelte

```diff
@@ -69,11 +69,12 @@
   let viewlets: Array<WithLookup<Viewlet>> = []
   let viewOptions: ViewOptions | undefined
 
-  $: spaceQuery = queryOptions?.filterBySpace === true && space !== undefined ? { space } : {}
-  $: _baseQuery = mergeQueries(mergeQueries(baseQuery ?? {}, viewlet?.baseQuery ?? {}), spaceQuery)
+  let isQueryLoaded = queryBuilder === undefined
+
+  $: _baseQuery = mergeQueries(baseQuery ?? {}, viewlet?.baseQuery ?? {})
   $: query = { ..._baseQuery }
   $: searchQuery = search === '' ? query : { ...query, $search: search }
-  $: resultQuery = searchQuery
+  $: resultQuery = isQueryLoaded ? searchQuery : undefined
 
   let options = viewlet?.options
 
```

Comment:
Please add queryBuilder as parameter to updateQuery, without it svelte will not rebuild query if it will change

Reply:
Thanks, fixed